docs/UserGuide.md

```diff
@@ -186,12 +194,16 @@ Find students by either their name, level or subject.
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
 
+***
+
 ### Listing All Students
 
 Shows a list of all students in the address book.
 
 **Format:** `list`
 
+***
+
 ### Tagging a Student
 
 Allows tagging a student by their level and subject. A student must have a level assigned before they can be tagged with a subject.
```

Comment:
Should we specify that we can tag a student with multiple subjects or is that implied through the ... in the command format?